src/citrine/_serialization/properties.py

```diff
@@ -360,7 +360,7 @@ def __init__(self,
 
     @property
     def underlying_types(self):
-        return list
+        return list, set, tuple
```

Comment:
I'd be open to it, (and tests pass with it), but I think I'd lean towards being more explicit/constrictive? We can always expand later if we find we want more flexibility.

Reply:
I'm having trouble tracking down the reference now, but I recall a discussion on the Python boards that made the point that duck-typing implies you should be testing for method compatibility (e.g., what `collections.abc` offers) rather than literal object inheritance.  Not blocking.